client/components/UserGroupRoles/RolesTable.jsx

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
```

Comment:
We dont need a duplicated RolesTable (there's an equal file in components/Roles folder)